decimal_test.go

```diff
@@ -745,7 +745,7 @@ func TestSizeof(t *testing.T) {
 	}
 }
 
-func TestJSONEncoding(t *testing.T) {
```

Comment:
Leave this entire test exactly as it was.

Reply:
Doing so will drop test coverage for `UnmarshalText()` and `MarshalText()`

Reply:
Oh, ok, I understand now. We'll use similar tests for testing marshaling/unmarshling text and JSON.